assets/scss/components/_actions-card.scss

```diff
@@ -1,5 +1,12 @@
 .actions-card {
-  display: none; // Hide the action card by default
+  display: none; // Hide the entire action card by default
```

Comment:
All the changes in the css should now support the Review actions. I dont think we'll need to do any more styling, they _should_ just work .... 🤞